Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOneV3/TrendMicroVisionOneV3.py

```diff
@@ -1775,177 +2204,253 @@ def submit_file_entry_to_sandbox(
+        "arguments": unwrap(resp.response).arguments,
     }
-    results = CommandResults(
+
+    return CommandResults(
         readable_output=tableToMarkdown(
-            TABLE_SUBMIT_FILE_ENTRY_TO_SANDBOX, message, removeNull=True
+            table_name[FILE_ENTRY_TO_SANDBOX_COMMAND],
+            message,
+            headerTransform=string_to_table_header,
+            removeNull=True,
         ),
         outputs_prefix="VisionOne.Submit_File_Entry_to_Sandbox",
-        outputs_key_field="entryId",
+        outputs_key_field="entry_id",
         outputs=message,
     )
-    return results
 
 
-def add_note(client: Client, args: Dict[str, Any]) -> Union[str, CommandResults]:
+def submit_urls_to_sandbox(
+    client: Client, args: Dict[str, Any]
+) -> Union[str, CommandResults]:
     """
-    Adds a note to an existing workbench alert
+    submit Urls to sandbox and send the result to demist war room
     :type client: ``Client``
-    :param client: client object to use http_request.
+    :param client: client object used to initialize pytmv1 client.
     :type args: ``dict``
     :param args: args object to fetch the argument data.
     :return: sends data to demisto war room.
     :rtype: ``dict`
     """
-    workbench_id = args.get(WORKBENCH_ID)
-    content = args.get(CONTENT)
+    # Required Params
+    urls: List[str] = []
+    # extract urls and add to urls list
+    for url in args[URLS]:
+        urls.append(url)
+    submit_urls_resp: List[Dict[str, Any]] = []
+    # Initialize pytmv1 client
+    v1_client = _get_client(APP_NAME, client.api_key, client.base_url)
+
+    # Make rest call
+    resp = v1_client.submit_urls_to_sandbox(*urls)
+    # Check if an error occurred during rest call
+    if _is_pytmv1_error(resp.result_code):
+        return_error(message=f"{unwrap(resp.errors)}", error=str(resp.errors))
+    for item in unwrap(resp.response).items:
+        submit_urls_resp.append(item.dict())
 
-    body = {"content": content}
-    query_params: Dict[str, Any] = {}
-    token = client.api_key
-    headers = {
-        "Authorization": "Bearer " + token,
-        "Content-Type": "application/json;charset=utf-8",
-    }
-    response = requests.post(
-        f"{client.base_url}" + ADD_NOTE_ENDPOINT.format(alertId=workbench_id),
-        params=query_params,
-        headers=headers,
-        data=json.dumps(body),
+    return CommandResults(
+        readable_output=tableToMarkdown(
+            table_name[URLS_TO_SANDBOX_COMMAND],
+            submit_urls_resp,
+            headerTransform=string_to_table_header,
+            removeNull=True,
+        ),
+        outputs_prefix="VisionOne.Submit_Urls_to_Sandbox",
+        outputs_key_field="id",
+        outputs=submit_urls_resp,
     )
-    response_code = response.status_code
-    location = response.headers.get("Location", "").split("/")
-    note_id = location[-1]
-    resp_msg = "success"
+
+
+def get_alert_details(
```

Comment:
the same here 
looks like `trendmicro-visionone-get-alert-details` not defined in the YML file

Reply:
added action details to YML.